### ISSUE TITLE
[Feature Request] Change text/prompt when using sending a picture

### ISSUE THREAD

**Issue by Sebyle** [SillyTavern/SillyTavern#728]
When using **Send a picture** feature, allow us to edit prompt/text of the picture sent.

**Comment by Woisek** [comment]
https://github.com/SillyTavern/SillyTavern/issues/571

**Comment by Woisek** [comment]
@Cohee1207 Sorry, my bad. Had it red completely wrong obviously. You are right.

**Comment by Cohee1207** [comment]
Released to main. Requires to enable it in extension settings first

**Comment by Woisek** [comment]
@Cohee1207 Could you please explain how it's done? I have enabled it, but clicking the brush icon still generates just an image without a prompt showing up before ... 🤔

**Comment by Cohee1207** [comment]
@Woisek This issue was for the captioning plugin. In both extensions (SD too) you need to enable the "Edit prompt/caption before sending" option

**Comment by Woisek** [comment]
@Cohee1207 
Yes, I have it enabled in both extensions. But still, clicking the brush icon sends what ever it is as prompt, directly off, no possibility to edit before. In dev it didn't work and now in main also not. I wasn't sure if this is active or not because of this, that's why I ask ... 
A bug maybe? 🤔

**Comment by Cohee1207** [comment]
Oops, I probably forgot that brush generation exists.

**Comment by Woisek** [comment]
> Oops, I probably forgot that brush generation exists.

😁

**Comment by Woisek** [comment]
> > Oops, I probably forgot that brush generation exists.

@Cohee1207 When can we expect this to be fixed? 😔
